### PR TITLE
Added Basic Pyhton Syntax tutorial

### PR DESCRIPTION
**Pull request by jon-chiou**
I've written a short tutorial on Basic Python Syntax in markdown. It covers the following topics:

1. An intro with comparison of Python print function to it's Java equivalent
2. The importance and use of whitespace (including examples of a function, while loop, and if/else conditional statement)
3. How to write single and multi-line comments
4. Variables and (basic, i.e. primative) data types

### CODE REVIEW

File: tutorials/python/posts/basic-syntax.md [hunk 1]
```diff
@@ -1 +1,116 @@
-Lorem ipsum dolor sit amet, consectetur adipisicing elit. At quisquam, doloremque laboriosam, architecto iure accusamus ut molestiae eos cum vel suscipit accusantium quidem natus, excepturi commodi. Esse quo quibusdam dolorem.
\ No newline at end of file
+#Basic Python Syntax
+
+###Introduction
+
+Python is reknown for having a streamlined syntax that makes it easy to read and write (relative to other programming languages).
```
Comment (darshil) [inline tutorials/python/posts/basic-syntax.md]: Please fix the spelling mistake.

reknown -> renown

File: tutorials/python/posts/basic-syntax.md [hunk 2]
```diff
@@ -1 +1,116 @@
-Lorem ipsum dolor sit amet, consectetur adipisicing elit. At quisquam, doloremque laboriosam, architecto iure accusamus ut molestiae eos cum vel suscipit accusantium quidem natus, excepturi commodi. Esse quo quibusdam dolorem.
\ No newline at end of file
+#Basic Python Syntax
```
Comment (darshil) [inline tutorials/python/posts/basic-syntax.md]: Have a space between # and word.

Refer to [this](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) for the correct Markdown syntax

**Review by darshil — COMMENTED** [review]
Please structure the content in a better manner, it does not meet the content style guidelines

Possible format:
- Getting Setup
- Introduction
- Variables
- Arrays
- Whitespace
- Conditionals
- Loops
- Methods
- Scope
- Comments

**Comment by jon-chiou** [comment]
@darshil I'll make these changes tonight. Where are the content style guidelines that I can reference?

**Comment by darshil** [comment]
@Chiouonthis Thanks! Unfortunately, we don't have content style guidelines just yet, but you can take a look at our other tutorials, just try to follow a similar style guideline to that.

Sorry for the inconvenience

File: tutorials/python/posts/basic-syntax.md (hunk 2 above)
Comment (Chiouonthis) [reply tutorials/python/posts/basic-syntax.md]: > @Chiouonthis Thanks! Unfortunately, we don't have content style guidelines just yet, but you can take a look at our other tutorials, just try to follow a similar style guideline to that.
> 
> Sorry for the inconvenience

Yeah, so I did use W3Schools as a reference: https://www.w3schools.com/python/python_syntax.asp

**Please don't ask someone to refer to guidelines that do not exist.** I'm happy to write up some of the additions you proposed, but I do not think it is right for you to say that I did not meet some standard that isn't even established.

**Comment by jon-chiou** [comment]
> @Chiouonthis Thanks! Unfortunately, we don't have content style guidelines just yet, but you can take a look at our other tutorials, just try to follow a similar style guideline to that.
> 
> Sorry for the inconvenience

Yeah, so I did use W3Schools as a reference: https://www.w3schools.com/python/python_syntax.asp

Please don't ask someone to refer to guidelines that do not exist. I'm happy to write up some of the additions you proposed, but I do not think it is right for you to say that I did not meet some standard that isn't even established.

**Review by darshil — APPROVED** [review]

**Comment by maaslalani** [comment]
@Chiouonthis Hey Jonathan, I'm going to merge your changes with a few additions and fixes. While this is merged you can work on the topics and change the overall structure of the tutorial. Great work! 👍

**Comment by jon-chiou** [comment]
@maaslalani Thanks! I added more content (PR #15) but did not re-arrange the order. I can keep working on this. Cheers!

**Pull request merged**